src/ParallelHashMap.h

```diff
@@ -676,16 +681,15 @@ void ParallelHashMap<K,V>::resize()
 
   /* reassign pointer */
   _table = new_map;
+#pragma omp flush(_table)
 
   /* release all locks */
-  #ifdef OPENMP
   for (size_t i=0; i<_num_locks; i++)
     omp_unset_lock(&_locks[i]);
-  #endif
 
   /* delete key and value list */
-  delete[] key_list;
-  delete[] value_list;
+  delete [] key_list;
+  delete [] value_list;
 
   /* wait for all threads to stop reading from the old table */
   for (size_t i=0; i<_num_threads; i++)
```

Comment:
Per the google c++, the while loop on line 692 should have `continue` after it instead of empty brackets.